lib/npg_tracking/illumina/run/long_info.pm

```diff
@@ -709,6 +711,36 @@ sub is_rapid_run_abovev2 {
   return $version && ($version > 2);
 }
 
+=head2 is_read_reverse_complement
+
```

Comment:
I find this method naming a bit confusing - I think its functionality is `_does_runinfo_indicate_i5_is_rev_complement` and it could be "hidden" as an attribute starting with `_` as I'd expect other code to only use `is_i5opposite` (and this "helper" to only be used in that).